R/installspec.r

```diff
@@ -14,11 +20,11 @@ installspec <- function(user = TRUE) {
             break
         }
     }
-    
+
     if (!found_binary)
-        stop('Jupyter or IPython 3.0 has to be installed but could neither run “jupyter” nor “ipython”, “ipython2” or “ipython3”.
-             (Note that “ipython2” is just IPython for Python 2, but still may be IPython 3.0)')
-    
+        stop('Jupyter or IPython 3.0 has to be installed but could neither run "jupyter" nor "ipython", "ipython2" or "ipython3".
+             (Note that "ipython2" is just IPython for Python 2, but still may be IPython 3.0)')
```

Comment:
oh and plain ascii quotes mean “string” to me as a programmer.

obviously i can interpret them contextually but i prefer to see `"string-literals"` marked up in monospace and with plain quotes, and “something quoted” in typographical quotes.